code/game/objects/items/crayons.dm

```diff
@@ -23,6 +25,8 @@
 	var/list/validSurfaces = list(/turf/simulated/floor)
 	var/times_eaten = 0 //How many times this crayon has been gnawed on
 	var/max_bites = 4 //How many times a crayon can be bitten before being depleted. You eated it
+	var/preset_message
+	var/preset_message_index = 0
```

Comment:
```suggestion
	/// How many times this crayon has been gnawed on
	var/times_eaten = 0
	/// How many times a crayon can be bitten before being depleted. You eated it
	var/max_bites = 4
	var/preset_message
	var/preset_message_index = 0
```
While we are here...
And i would suggest documenting the vars you added too.

Reply:
Done, PTAL